### PR TITLE
Mainnet upgrade 1.3.4

### PR DESCRIPTION
**Pull request by mrkara**

### CODE REVIEW

File: docs/aws/setup-mainnet-validator-from-scratch.md [hunk 1]
```diff
@@ -146,7 +146,7 @@ cd casper-node/
 > installed.
 
 ```
-git checkout release-1.2
+git checkout release-1.3.2
```
Comment (mssteuer) [inline docs/aws/setup-mainnet-validator-from-scratch.md:149]: @mrkara should this not be 1.3.4?

File: docs/aws/setup-testnet-validator-from-scratch.md [hunk 2]
```diff
@@ -168,7 +168,7 @@ cd casper-node/
 > installed.
 
 ```
-git checkout release-1.2
+git checkout release-1.3.2
```
Comment (mssteuer) [inline docs/aws/setup-testnet-validator-from-scratch.md:171]: @mrkara should this not be 1.3.4?

**Review by mssteuer — CHANGES_REQUESTED** [review]
@mrkara im seeing a bunch of 1.3.2 instead of 1.3.4 for contract building etc. But I'm reviewing from my phone so may be seeing things incorrectly. Please review again and let me know if/when it's good to go.

File: docs/aws/setup-testnet-validator-from-scratch.md (hunk 2 above)
Comment (mrkara) [reply docs/aws/setup-testnet-validator-from-scratch.md:171]: It's not a typo. There is no 1.3.4 branch (yet).

File: docs/aws/setup-mainnet-validator-from-scratch.md (hunk 1 above)
Comment (mrkara) [reply docs/aws/setup-mainnet-validator-from-scratch.md:149]: It's not a typo. There is no 1.3.4 branch (yet).

**Comment by mrkara** [comment]
@mssteuer The 1.3.2 values are intentional. There is no 1.3.4 branch (yet). It's good to go. :)

**Pull request merged**